dashboard/db/schema.rb

```diff
@@ -918,6 +929,7 @@
   add_foreign_key "authored_hint_view_requests", "users"
   add_foreign_key "hint_view_requests", "users"
   add_foreign_key "level_concept_difficulties", "levels"
+  add_foreign_key "pd_enrollments", "school_districts"
```

Comment:
Good question.  I like having a foreign key constraint just to maintain some data integrity.  But perhaps we could add an additional "deleted" column to school_districts if/when we start removing districts?